DistTasks.yml

```diff
@@ -0,0 +1,167 @@
+version: "3"
```

Comment:
We can actually use the same one used in Arduino Lint and Arduino CLI really but it would be overkill in my opinion.

The main difference between this project and the other two is that we don't use CGO in here so to crosscompile we just need to set some env vars and that's it, the other two instead use it so it's best to use Docker containers to ease the handling of the necessary dependencies to enable crosscompilation with CGO.

I think we can take the Arduino CLI `DistTasks.yml` and use it in here with just a small modification in the package creation since here we need to include the `firmwares` folder.

Reply:
Thanks for the explanation!